osidb/models.py

```diff
@@ -1501,6 +1501,7 @@ class Source(models.TextChoices):
         """
 
         NVD = "NVD"
+        FEED = "FEED"
```

Comment:
I do not think `Feed` is a good name for a source. It is only a general way to describe a subset of sources. I would prefer to use the names of parsed files as sources.

Reply:
Do you want as many sources as there are sources in the Feed collector's config file or did I misunderstand?

Reply:
That means that when `confix.yaml` is modified with a new feed, the Source class will have to be modified as well. Is that still what you meant?

Reply:
Yes.

Reply:
Clear now. Thank you.

Reply:
Done.